e2e/playwright/tests/shared/deploy-app.ts

```diff
@@ -1,5 +1,7 @@
 export const deployApp = async (page, expect) => {
-  await expect(page.getByRole('button', { name: 'Add node', exact: true })).toBeVisible();
+  await expect(page.getByText('Optionally add nodes to the cluster'),).toBeVisible();
+  const joinCommand = await page.locator('.react-prism.language-bash').first().textContent();
+  console.log(`{"command":"${joinCommand}"}`);
```

Comment:
we only log this so it's detected by our e2e tests. how about running one or two validations that the command is visible instead?